crates/atuin/src/command/client.rs

```diff
@@ -120,9 +121,20 @@ impl Cmd {
     ) -> Result<()> {
         let filter =
             EnvFilter::from_env("ATUIN_LOG").add_directive("sqlx_sqlite::regexp=off".parse()?);
-
+        let log_dir = atuin_common::utils::state_dir().join("logs");
+        // Ensure the log directory exists
+        tokio::fs::create_dir_all(&log_dir)
+            .await
+            .wrap_err("could not create log directory")?;
+        let appender = RollingFileAppender::builder()
+            .filename_prefix("atuin.log")
+            .rotation(Rotation::DAILY)
+            .max_log_files(7)
```

Comment:
I would probably make this configurable if we want to keep the rotating appender around. If we do, should this just be an env var or also exposed in settings?

Reply:
I added an env var for now just in case. Easy enough to pop of or modify depending on feedback!